packages/exo/src/exo-tools.js

```diff
@@ -263,8 +264,17 @@ export const defendPrototype = (
   let methodGuards;
   if (interfaceGuard) {
     assertInterfaceGuard(interfaceGuard);
-    const { interfaceName, methodGuards: mg, sloppy = false } = interfaceGuard;
-    methodGuards = mg;
+    const {
+      interfaceName,
+      methodGuards: mg,
+      symbolMethodGuards,
+      sloppy = false,
+    } = interfaceGuard;
+    methodGuards = harden({
+      ...mg,
+      ...(symbolMethodGuards &&
+        fromEntries(getCopyMapEntries(symbolMethodGuards))),
+    });
```

Comment:
Nice!

Given this, my preference to use [...x, ...y] above is stronger. Now I will request that change ;)